hive/src/main/java/org/apache/iceberg/parquet/AdaptHiveParquetConversions.java

```diff
@@ -85,43 +84,48 @@ static <T> Literal<T> fromParquetPrimitive(Type type, PrimitiveType parquetType,
     }
   }
 
-  static Function<Object, Object> converterFromParquet(PrimitiveType parquetType, Type icebergType) {
+  static Function<Object, Object> converterFromParquet(
+      PrimitiveType parquetType, Type icebergType) {
 
-    //Change For Arctic:Adapt int 96 and bytes string
-    //int96
+    // Change For Arctic:Adapt int 96 and bytes string
+    // int96
     if (parquetType.getPrimitiveTypeName() == PrimitiveType.PrimitiveTypeName.INT96) {
       return binary -> {
-        final ByteBuffer byteBuffer = ByteBuffer.wrap(((Binary) binary).getBytes()).order(ByteOrder.LITTLE_ENDIAN);
+        final ByteBuffer byteBuffer =
+            ByteBuffer.wrap(((Binary) binary).getBytes()).order(ByteOrder.LITTLE_ENDIAN);
         final long timeOfDayNanos = byteBuffer.getLong();
         final int julianDay = byteBuffer.getInt();
-        Instant instant = Instant
-            .ofEpochMilli(TimeUnit.DAYS.toMillis(julianDay - 2_440_588L))
-            .plusNanos(timeOfDayNanos);
+        Instant instant =
+            Instant.ofEpochMilli(TimeUnit.DAYS.toMillis(julianDay - 2_440_588L))
+                .plusNanos(timeOfDayNanos);
 
         if (!((Types.TimestampType) icebergType).shouldAdjustToUTC()) {
-          //iceberg org.apache.iceberg.expressions.Literals resolve timestamp without tz use UTC, but in fact it is
+          // iceberg org.apache.iceberg.expressions.Literals resolve timestamp without tz use UTC,
+          // but in fact it is
           // local time zone
```

Comment:
In before this is via press Enter to format, but now it looks a bit strange after `spotless:apply`.

Reply:
fixed.